fe/fe-core/src/main/java/org/apache/doris/planner/OriginalPlanner.java

```diff
@@ -572,50 +572,14 @@ private void pushSortToOlapScan() {
      * column unique id for `A` and `B` will put into outputColumnUniqueIds.
      *
     */
+    // this opt will only work with nereidsPlanner
     private void pushOutColumnUniqueIdsToOlapScan(PlanFragment rootFragment, Analyzer analyzer) {
         Set<Integer> outputColumnUniqueIds = new HashSet<>();
-        ArrayList<Expr> outputExprs = rootFragment.getOutputExprs();
-        for (Expr expr : outputExprs) {
-            if (expr instanceof SlotRef) {
-                if (((SlotRef) expr).getColumn() != null) {
-                    outputColumnUniqueIds.add(((SlotRef) expr).getColumn().getUniqueId());
-                }
-            }
-        }
+        // add '-1', make compatible for nereidsPlanner
```

Comment:
change comment to "add '-1' to avoid the optimization of xxxxxxx, because yyyyy"

Reply:
updated